pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: dartdoc
 # Also update the `version` field in lib/dartdoc.dart.
-version: 0.14.1
+version: 0.14.2-dev
```

Comment:
add entry to changelog if you like but I prefer to avoid updating the version number for non-breaking changes until release.

Reply:
I still think this is poor CM practice but if it is standard for Dart package development, we can do this.  A better way might be to put this in a separate PR.

Reply:
I'd like to see future -dev changes made in separate PRs to make reverts easier.

But if we're going to keep this one, the version number in lib/dartdoc.dart should also change.

Reply:
Happy to do a rebase cleanup...if you have no other comments